rotkehlchen/db/upgrades/v45_v46.py

```diff
@@ -53,4 +53,14 @@ def _refresh_icons(write_cursor: 'DBCursor') -> None:
             icon_path = icons_dir / f'{urllib.parse.quote_plus(identifier)}_small.png'
             icon_path.unlink(missing_ok=True)
 
+    @progress_step(description='Moving EVM event extra data to the history_events table.')
+    def _move_extra_data(write_cursor: 'DBCursor') -> None:
+        write_cursor.execute('ALTER TABLE history_events ADD COLUMN extra_data TEXT;')
+        write_cursor.execute(
+            'UPDATE history_events SET extra_data = '
+            '(SELECT extra_data FROM evm_events_info '
+            'WHERE evm_events_info.identifier = history_events.identifier);',
+        )
```

Comment:
I've added a check for the contents of an evm event that already is present in the db, and also a check for a plain history entry. As noted in the comments in the code, the history entry has to be inserted manually instead of creating a HistoryEvent or EthStakingEvent class, since those classes now expect there to be an `extra_data` column on the `history_events` table, and would not work correctly before the upgrade is run.